astro/src/content/docs/get-started/core-concepts/integration-points.mdx

```diff
@@ -77,22 +78,28 @@ Recommended storage mechanisms include:
 
 * sending the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
 * storing it server side, in a session. The session is typically managed by a framework, and ideally adheres to the same cookie storage recommendations. Learn more about [server side sessions](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html). Please consult your framework documentation around securing sessions and data in sessions.
```

Comment:
This may be a good opportunity to clean up uses of "server side" vs "server-side" in these files. When used as an adjective, the term should be hyphenated.
```suggestion
* storing it server side, in a session. The session is typically managed by a framework, and ideally adheres to the same cookie storage recommendations. Learn more about [server-side sessions](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html). Please consult your framework documentation around securing sessions and data in sessions.
```